static/js/notifications.js

```diff
@@ -402,6 +402,17 @@ function message_is_notifiable(message) {
         return false;
     }
 
+    // The notification should only be displayed once. To prevent multiple
+    // tabs from opening a notification each for their own, we store the
+    // notification ID in localStorage and check if a notification was
+    // already opened.
+    if (typeof(Storage) !== "undefined") {
```

Comment:
I am actually not sure how I should use this function. Calling supports_localstorage() directly doesn't work. In `localstorage.js`, the function is defined like this: `exports.supported = function supports_localstorage() {
`
However, calling `exports.supported()` doesn't work either.